pyannote/audio/interactive/answers2file.py

```diff
@@ -0,0 +1,56 @@
+# Que se passe-t-il si pas de Speech ?
```

Comment:
This file `answers2file.py` should be removed from the PR for now because it is only used in `update` that we will not do in this PR.